### ISSUE TITLE
Ansible Runner task not working on OpenShift

### ISSUE THREAD

**Issue by piyush-garg** [tektoncd/catalog#495]
# Expected Behavior

Ansible task should work on OpenShift

# Actual Behavior

Ansible task failing to run on Openshift with following error

```
/-\|/-\|/-\+ ansible-runner run -p list-pods.yml kubernetes
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that
the implicit localhost does not match 'all'

PLAY [localhost] ***************************************************************

TASK [Gathering Facts] *********************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: KeyError: 'getpwuid(): uid not found: 1000870000'
fatal: [localhost]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}

PLAY RECAP *********************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
```

# Steps to Reproduce the Problem

1. Apply the ansible-runner task and git-clone task.
2. Create the resources available in tests directory
3. Create the pipeline and pipelinerun available in tests directory

# Additional Info

Pipeline version: 0.15.2

/kind bug

**Comment by kameshsampath** [comment]
Fixed via https://github.com/tektoncd/catalog/pull/496

**Comment by piyush-garg** [comment]
Fixed by #496 

/close
